packages/eslint-config/index.js

```diff
@@ -0,0 +1 @@
+module.exports = require('./.eslintrc.json');
```

Comment:
Is this `require` mandatory ? 
I find it weird that there is one for the JSON files but not for the YAML ones ?

Reply:
This wrapper allows use tu write `@talend/eslint-config` instead of `@talend/eslint-config/.eslintrc.json`.
The others are yaml, so no possible js wrapper (especially with the extends of other yaml files). Moreover, they are named profiles, so having them directly or with a wrapper doesn't change anything int your app eslintrc `extends` value.